weatherman.py

```diff
@@ -0,0 +1,109 @@
+"""
+Weatherman
+This module generates yearly reports on weather data collected from  Dec 1996 - May 2011.
+Report no. 1 gives the yearly max/min temperature and max/min humidity.
+Report no. 2 gives the hottest day of the year and the temperature on that day.
+"""
+
+import os
+import sys
+
+import argparse
+
+
+max_temp_ = {}
```

Comment:
why is there a underscore at the end?

Reply:
**Previous comment:** Changed to max_temp_. Added the extra '_' to avoid confusion, as suggested by PEP-8, with the 'max_temp' variable. Should I change this?
Also changed coldest_day and hottest_date.
**Revision:** PEP-8 suggests this for reserved words, I was wrong. Just wanted to avoid conflict with the variable name 'max_temp'.
Changed.